pallets/avn/src/tests/session_handler_tests.rs

```diff
@@ -1,167 +1,273 @@
 // Copyright 2022 Aventus Network Services (UK) Ltd.
 
 use crate::mock::*;
-use sp_avn_common::event_types::Validator;
```

Comment:
Can we reinstate this use, to avoid the line wrap on line 7 to improve readability